tools/clickbench/cbqueries.csv

```diff
@@ -0,0 +1,40 @@
+"SearchPhrase != """" | stats count as c, sum(IsRefresh), avg(ResolutionWidth) by SearchEngineID, ClientIP | sort -c | head 10",300
+"SearchPhrase != """" | stats count as c, sum(IsRefresh), avg(ResolutionWidth) by WatchID, ClientIP | sort -c | head 10",300
+"* | stats count as c, sum(IsRefresh), avg(ResolutionWidth) by WatchID, ClientIP | sort -c | head 10",500
+"* | stats count as c by URL | sort -c | head 10",320
+"* | eval n = 1 | stats count as c by n, URL | sort -c | head 10",5500
+"* | eval cp = ClientIP-1, cp2 = ClientIP-2, cp3 = ClientIP-3 | stats count as c by ClientIP, cp, cp2, cp3 | sort -c | head 10",7500
+"CounterID = 62 AND DontCountHits = 0 AND IsRefresh = 0 AND URL != """" | where strptime(EventDate,""%Y-%m-%d"") >= strptime(""2013-07-01"", ""%Y-%m-%d"") AND strptime(EventDate,""%Y-%m-%d"") <= strptime(""2013-07-31"", ""%Y-%m-%d"")  | stats count as PageViews by URL | sort -PageViews | head 10",5600
+"CounterID = 62 AND DontCountHits = 0 AND IsRefresh = 0 AND Title != """" | where strptime(EventDate,""%Y-%m-%d"") >= strptime(""2013-07-01"", ""%Y-%m-%d"")  AND strptime(EventDate,""%Y-%m-%d"") <= strptime(""2013-07-31"", ""%Y-%m-%d"") | stats count as PageViews by Title | sort -PageViews | head 10",5200
+"CounterID = 62 AND IsRefresh = 0 AND IsLink != 0 AND IsDownload = 0 | where strptime(EventDate,""%Y-%m-%d"") >= strptime(""2013-07-01"", ""%Y-%m-%d"")  AND strptime(EventDate,""%Y-%m-%d"") <= strptime(""2013-07-31"", ""%Y-%m-%d"") | stats count as PageViews by URL | sort -PageViews | head 1010 | tail 10 | tail 10",700
+"CounterID = 62 AND IsRefresh = 0 | where strptime(EventDate,""%Y-%m-%d"") >= strptime(""2013-07-01"", ""%Y-%m-%d"") AND strptime(EventDate,""%Y-%m-%d"") <= strptime(""2013-07-31"", ""%Y-%m-%d"") | eval Src=if(SearchEngineID=0 AND AdvEngineID=0, Referer, """") | rename URL as Dst | stats count as PageViews by TraficSourceID, SearchEngineID, AdvEngineID, Src, Dst | sort -PageViews | head 1010 | tail 10 | tail 10",8000
+"CounterID = 62  AND IsRefresh = 0 AND RefererHash = ""3594120000172545465"" | where TraficSourceID in(-1, 6) | where strptime(EventDate,""%Y-%m-%d"") >= strptime(""2013-07-01"", ""%Y-%m-%d"") AND strptime(EventDate,""%Y-%m-%d"") <= strptime(""2013-07-31"", ""%Y-%m-%d"") | stats count as PageViews by URLHash, EventDate | sort -PageViews | head 110 | tail 10 | tail 10",1200
+"CounterID = 62  AND IsRefresh = 0 AND DontCountHits = 0 AND URLHash = ""2868770270353813622"" | where strptime(EventDate,""%Y-%m-%d"") >= strptime(""2013-07-01"", ""%Y-%m-%d"") AND strptime(EventDate,""%Y-%m-%d"") <= strptime(""2013-07-31"", ""%Y-%m-%d"") | stats count as PageViews by WindowClientWidth, WindowClientHeight | sort -PageViews | head 10010 | tail 10 | tail 10",1200
```

Comment:
there are only 40 rows here, we should include the `"null"` queries here so that we can keep track of the query nums, and eventually we will start supporting those queries, so then we can do a line by line match of each query against the original queries.spl file

Reply:
addressed in the latest commit.